ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -55,7 +54,7 @@
   public static long GENESIS_SLOT = 4294967296L; // 2^32
   public static long GENESIS_EPOCH = slot_to_epoch(GENESIS_SLOT);
   public static long GENESIS_START_SHARD = 0;
-  public static UnsignedLong FAR_FUTURE_EPOCH = UnsignedLong.MAX_VALUE;
+  public static long FAR_FUTURE_EPOCH = Long.MAX_VALUE;
```

Comment:
This is not the right value, strictly speaking (it should be `2^64 -1` which is acually `-1` in terms of bits). We need to work out how to deal with this - internally we can use anything, but we need to serialise and deserialise this correctly in case we send/receive it on the wire.